kong/pdk/node.lua

```diff
@@ -18,6 +19,7 @@ local shared = ngx.shared
 local C             = ffi.C
 local ffi_new       = ffi.new
 local ffi_str       = ffi.string
+local get_lmdb_info = lmdb.get_env_info
```

Comment:
We have the different name `get_env_info` in statsd, should we give them the same name?